### PR TITLE
Not a real commit

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Just making a PR so I can leave comments on the text.

### CODE REVIEW

File: aip/0194.md [hunk 1]
```diff
@@ -25,6 +26,7 @@ Most requests can either be categorized as **retryable**, **transactional**, or
 unintended state changes. Most CRUD RPCs which take a unique identifier are
 considered retryable (see more below in "Idempotency and pragmaticism").
```
Comment (johanbrandhorst) [inline aip/0194.md:27]: Should this me `pragmatism`? I found only an [obscure branch of philosophy](https://en.wikipedia.org/wiki/Pragmaticism) for `pragmaticism`.

File: aip/0194.md [hunk 2]
```diff
@@ -25,6 +26,7 @@ Most requests can either be categorized as **retryable**, **transactional**, or
...
 
+
 **Transactional** requests are requests which are part of a transactions. These
```
Comment (johanbrandhorst) [inline aip/0194.md:30]:
```suggestion
**Transactional** requests are requests which are part of a transaction. These
```

File: aip/0194.md [hunk 3]
```diff
@@ -47,6 +49,7 @@ connection will become available (soon).
 Streaming requests can be categorized as **server streaming**,
 **client streaming**, and **bidirectional streaming**.
```
Comment (johanbrandhorst) [inline aip/0194.md:50]: This feels like it should be either
```suggestion
**client streaming**, or **bidirectionally streaming**.
```
or
```suggestion
Streaming requests can use **server streaming**,
**client streaming**, or **bidirectional streaming**.
```

File: aip/0194.md [hunk 4]
```diff
@@ -57,6 +60,7 @@ data being streamed.
 **Bidirectional streaming** request retryability is highly dependent on the type
 of data being streamed.
```
Comment (johanbrandhorst) [inline aip/0194.md:61]: Maybe
```suggestion
of data being exchanged.
```
To differentiate it from the previous line.

File: aip/0194.md [hunk 5]
```diff
@@ -57,10 +60,12 @@ data being streamed.
...
 
+
 ## Codes that should never be retried
 
 - `OK`: The request succeeded.
 - `CANCELLED`: A user can cancel a request, which we should honour.
```
Comment (johanbrandhorst) [inline aip/0194.md:67]: General comment for the document; Is using the first person correct here? Alternatively
```suggestion
- `CANCELLED`: A user can cancel a request, which should be honoured.
```

File: aip/0194.md [hunk 6]
```diff
@@ -122,6 +127,7 @@ wonder: how did we receive this result?
 The definition of idempotency is unfortunately vague in this regard. Idempotent
```
Comment (johanbrandhorst) [inline aip/0194.md:127]: What definition? Reference a source.

File: aip/0194.md [hunk 7]
```diff
@@ -214,6 +220,7 @@ first failed Create request, it unexpectedly returned `ALREADY_EXISTS`.
 However, it is easy to see that _one of_ these requests would have to receive
 `ALREADY_EXISTS`, even if no retrying happening.
```
Comment (johanbrandhorst) [inline aip/0194.md:221]:
```suggestion
 `ALREADY_EXISTS`, even if no retrying is happening.
```

**Review by johanbrandhorst — COMMENTED** [review]
I feel like this is missing a conclusion. The recommendation seems pretty obvious from reading it, so just make a nice TL;DR at the end with a nice sign-off. All the technical stuff looks good!

**Comment by johanbrandhorst** [comment]
@jadekler

**Comment by johanbrandhorst** [comment]
Oh, and don't merge this stuff, I just wanted to leave comments on the original doc.

**Comment by jeanbza** [comment]
tyvm!!

**Pull request closed without merging**